plugins/pluggable/pluggable.jsx

```diff
@@ -10,9 +10,14 @@ export default class Pluggable extends React.PureComponent {
     static propTypes = {
 
         /*
-         * Should be a single overridable React component
+         * Should be a single overridable React component. One of this or overrideName is required
          */
-        children: PropTypes.element.isRequired,
+        children: PropTypes.element,
+
+        /*
+         * Override the component to be plugged. One of this or children is required
+         */
+        overrideName: PropTypes.string,
```

Comment:
Not particularly happy with this name, open to suggestions

Reply:
`pluginName`, `pluggableName`, `pluginReference`?

Reply:
I'm bad with names cant help :(

Reply:
> There are only two hard things in Computer Science: cache invalidation and naming things.

-- Phil Karlton